assets/scss/mixins/main.scss

```diff
@@ -42,16 +42,19 @@
 
 /*
  * Define common styles for all the waves in the page
+ $bottom: define how far the wavy line will be from the element bottom
+ $height: define how thick the line will be
+ $bg-size-height: define how thick the line will be
  */
-@mixin wave($bottom) {
+@mixin wave($bottom, $height: 24px, $bg-size-height: 20px) {
   position: relative;
 
   &::after {
     position: absolute;
     bottom: $bottom;
     left: 0;
     width: 100%;
-    height: 24px;
+    height: $height;
```

Comment:
I understand the main reason for this change is adapting to different font sizes. What about using `em` instead of `px` to make it relative to the text size?

Reply:
Not really, the aim of this change is to have a wavy underline for the links and a wavy underline for decoration, the first one thinner than the second one.

Is not a straight way change, in this case we need to be in control of the size, so it's better to use `px`.

There's a section of the CSS for JS course where the units are very well explained, this is the summary, it helps a lot! :)
<img width="800" alt="Screenshot 2021-05-26 at 17 11 56" src="https://user-images.githubusercontent.com/39853718/119685346-92368700-be45-11eb-934b-d01f26588fd2.png">